src/pkg/backup/details/iteminfo.go

```diff
@@ -20,16 +20,17 @@ type ItemType int
 // Additionally, any itemType directly assigned a number should not be altered.
 // This applies to  OneDriveItem and FolderItem
 const (
-	UnknownType ItemType = iota // 0, global unknown value
+	UnknownType ItemType = 0
 
 	// Exchange (00x)
-	ExchangeContact
-	ExchangeEvent
-	ExchangeMail
+	ExchangeContact ItemType = 1
+	ExchangeEvent   ItemType = 2
+	ExchangeMail    ItemType = 3
+
 	// SharePoint (10x)
-	SharePointLibrary ItemType = iota + 97 // 100
```

Comment:
I believe `SharePointLibrary` will be 101

Reply:
Arg, the comments are off!?

Reply:
yup, you were right.  Good catch!

Reply:
I've compared them betwen main and the branch.  Plus wrote a little unit test on each end that I deleted. They are, with absolute confidence, the same values.